Frameworks/UIKit/UITouch.mm

```diff
@@ -225,7 +226,14 @@ - (CGPoint)locationInView:(UIView*)viewAddr {
     if (viewAddr != 0) {
         UIWindow* window = [viewAddr window];
         if (window != nil) {
-            ret = [window _convertPoint:location fromView:window toView:viewAddr];
+            UIWindow* fromWindow = window;
+
+            if (GetCACompositor()->IsRunningAsFramework()) {
+                // In framework mode the parent UIWindow is not visible
+                fromWindow = nil;
+            }
+
+            ret = [window _convertPoint:location fromView:fromWindow toView:viewAddr];
```

Comment:
>window _convertPoint [](start = 19, length = 20)

This code may look strange. Why are we calling a method on a window that we just declared is not relevant? Well, you see, the _convertPoint method actually makes no sense.